dockstore-webservice/src/main/java/io/dockstore/webservice/permissions/PermissionsInterface.java

```diff
@@ -169,17 +169,17 @@
     }
 
     /**
-     * Checks if a the email is an "original owner" of a workflow, and throws a
+     * Checks if a the username is an "original owner" of a workflow, and throws a
      * {@link CustomWebApplicationException} if it is. To be used as a check so
-     * that original owners never remove themselves as owners.
+     * that original owners can never be removed as owners.
      *
      *
-     * @param email
+     * @param username
      * @param workflow
      */
-    static void checkUserNotOriginalOwner(String email, Workflow workflow) {
-        if (workflow.getUsers().stream().anyMatch(u ->  email.equals(u.getUsername()))) {
-            throw new CustomWebApplicationException("", HttpStatus.SC_BAD_REQUEST);
+    static void checkUserNotOriginalOwner(String username, Workflow workflow) {
+        if (workflow.getUsers().stream().anyMatch(u ->  username.equals(u.getUsername()))) {
```

Comment:
Might be worth code commenting/warning above, this check is done via username. 
So things might still potentially break if we, say, allow username changes. 
(Not proposing a change, just thinking through our limitations)